py3status/modules/group.py

```diff
@@ -0,0 +1,193 @@
+        (default True)
+    format: Format for module output. (default "GROUP: {output}")
+
+
+Format of status string placeholders:
+    {output} Output of current active module
+
+Example:
+
+```
+# Create a disks group that will show space on '/' and '/home'
+# Change between disk modules every 30 seconds
+...
+order += "group disks"
+...
+
+group disks {
+    cycle = 30
+    format = "Disks: {output}"
+
+    disk "/" {
+        format = "/ %avail"
+    }
+
+    disk "/home" {
+        format = "/home %avail"
+    }
+}
+```
+
+@author tobes
+"""
+
+from time import time
+
+
+class Py3status:
+    # available configuration parameters
+    button_next = 4
+    button_prev = 5
+    color = None
+    cycle = 0
+    fixed_width = True
+    format = "GROUP: {output}"
+
+    class Meta:
+        include_py3_module = True
+
+    def __init__(self):
+        self.items = []
+        self.active = 0
+        self.initialized = False
+
+    def _init(self):
+        self._cycle_time = time() + self.cycle
+        self.py3_wrapper = self.py3_module.py3_wrapper
+        self.initialized = True
+
+    def _get_output(self):
+        if not self.items:
+            return
+        if not self.fixed_width:
+            return self._get_current_output(self.active)
+        current = None
+        widths = []
+        for i in range(len(self.items)):
+            output = self._get_current_output(i)
+            if not output:
+                continue
+            if i == self.active:
+                current = output
+            widths.append(len(output['full_text']))
+        width = max(widths)
+        current['full_text'] += ' ' * (width - len(current['full_text']))
+        return current
+
+    def _get_current_output(self, item):
+        output = None
+        current = self.items[item]
+        py3_wrapper = self.py3_wrapper
+        if current in py3_wrapper.modules:
+            for method in py3_wrapper.modules[current].methods.values():
+                output = method['last_output']
+        else:
+            if py3_wrapper.i3status_thread.config.get(current,
+                                                      {}).get('response'):
+                output = (
+                    py3_wrapper.i3status_thread.config[current]['response'])
+        return output
+
+    def _get_current_module_name(self):
+        if not self.items:
+            return
+        return self.items[self.active]
+
+    def _get_current_module(self):
+        if not self.items:
+            return
+        current = self.items[self.active]
+        py3_wrapper = self.py3_wrapper
+        if current in py3_wrapper.modules:
+            return py3_wrapper.modules[current]
+        else:
+            return py3_wrapper.config.get(current)
+
+    def _next(self):
+        self.active = (self.active + 1) % len(self.items)
+
+    def _prev(self):
+        self.active = (self.active - 1) % len(self.items)
+
+    def output(self, i3s_output_list, i3s_config):
```

Comment:
I think we should rename it to `group` or `group_output` so it shows correctly on the logs.

`method output returned {'instance': 'tz', 'name': 'group', 'full_text': 'LA 01:56', 'cached_until': 1458550600.423212}`